### DISCUSSION TITLE
Sidebar

### DISCUSSION THREAD

**Discussion by azhar7** [cosscom/coss#24] [Suggestions]
Sidebar nav either for a menu or a list. Items in both can have dropdowns of their own for navigation. 
Also, it might be good to have sidebar support LTR or RTL.

**Comment by AimadYahyaoui** [comment]
like shadcn/ui [https://ui.shadcn.com/docs/components/sidebar](https://ui.shadcn.com/docs/components/sidebar)

**Reply by azhar7** [reply]
Yes.

**Comment by corysimmons** [comment]
Lot of upvotes on this. :D

**Comment by Faizanahmedsy** [comment]
That would be great.

**Comment by vineyrawat** [comment]
It's already there in Shadcn why do you want it here?

**Reply by corysimmons** [reply]
The whole point of this library is to wrap and re-skin shadcn components.
